ai-worker/app/resumes/schemas.py

```
@@ -0,0 +1,49 @@
+    class Config:
+        json_schema_extra = {
+            "example": {
+                "id": 1,
+                "type": "RESUME",
+                "data": {
+                    "experience": [],
+                    "skills": ["Python", "Java"],
+                    "ai_description": "..."
+                }
+            }
+        }
```

Ruff (0.14.11)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)